### PR TITLE
Adding Pizzly - API Integrations Manager

### PR DESCRIPTION
**Pull request by Frenchcooc**
Submitting [Pizzly](https://github.com/bearer/pizzly) - The simplest, fatest way to integrate your app with an OAuth API 😋

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,6 +190,7 @@ Table of Contents
   * [OpenAPI3 Designer](https://interaction-design.co.za/openapidesigner/) — Visually create Open API 3 definitions for free
   * [parsehub.com](https://parsehub.com/) — Extract data from dynamic sites, turn dynamic websites into APIs, 5 projects free
   * [Pixela](https://pixe.la/) - Free daystream database service. All operations are performed by API. Visualization with heat maps and line graphs is also possible.
+  * [Pizzly](https://github.com/bearer/pizzly/) - Simplest, fatest way to integrate an app with an OAuth API.
```
Comment (ripienaar) [inline README.md:193]: whats the SaaS offering here?

Comment (Frenchcooc) [reply README.md]: Hi @ripienaar. It's an open-source project that makes it easy to integrate an application (SPA) with an OAuth API.

Comment (ripienaar) [reply README.md]: Yes, but this list is for SaaS offerings, so I am curious what the SaaS angle is here?

Comment (Frenchcooc) [reply README.md]: Pizzly is a self-hosted SaaS. You deploy it on Heroku for example and then you define which API you want to use and _voilà_. You're ready to use it from your frontend or backend. [Here are a few screenshots of the dashboard (i.e. the SaaS)](https://github.com/Bearer/Pizzly/wiki/Reference-:-Dashboard).

Comment (ripienaar) [reply README.md]: There's no self-hosted saas, sorry. not suited to the list.

**Pull request closed without merging**